cogs/fixtures.py

```diff
@@ -306,14 +306,29 @@ def findFixtures(matches, number: int):
             continue
         #the rest can be split by \n\n\n
         matchArray = match.text.split('\n\n\n')
+        matchYear = matchArray[0].split('\n\n')[1].strip()
+        #remove anything in the string that is not a number
+        matchYear = ''.join(filter(str.isdigit, matchYear))
+        matchYear = datetime.strptime(matchYear, '%Y')
         matchArray.pop(0) #pops the first element which is the month and year
         matchArray.pop(-1) #pops the last element which is an empty string
         for arr in matchArray:
             #check if the match is in the past, if it is, skip it
             #example arr: Wed Aug 2 - 18:00\n\n  Arsenal\n          \n 1 - 1\n\n  Monaco\n          \nEmirates Cup    
+            if "(Date and time TBC)" in arr:
+                arr = arr.replace("(Date and time TBC)          ", "\n") #women's fixtures and results have this for some reason, replacing it with newline
+            if "Time TBC" in arr:
+                arr = arr.replace("Time TBC          ", "00:00 \n") #some mens fixtures have this, replacing it with midnight until we get the time

```

Comment:
Would it make more sense to default to 3:00pm, as that's what they do pre-TV assignments?

Reply:
great idea, let me change that real quick